ddtrace/contrib/pynamodb/patch.py

```diff
@@ -0,0 +1,81 @@
+"""
+Trace queries to botocore api done via a pynamodb client
+"""
+
+import os
+from ddtrace.vendor import wrapt
+from ddtrace import config
+import pynamodb.connection.base
+
+
+from ...constants import ANALYTICS_SAMPLE_RATE_KEY, SPAN_MEASURED_KEY
+from ...pin import Pin
+from ...ext import SpanTypes
+from ...utils.formats import deep_getattr
+from ...utils.wrappers import unwrap
+from .. import trace_utils
+
+# Pynamodb connection class
+_PynamoDB_client = pynamodb.connection.base.Connection
+
+config._add("pynamodb", {"service_name": os.getenv("DD_PYNAMODB_SERVICE") or "pynamodb",})
```

Comment:
```suggestion
config._add("pynamodb", {"_default_service": "pynamodb",})
```

This was also part of the service name changes made recently 🙂